kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -20,17 +44,31 @@
 
   const Kolibri = require('kolibri');
   module.exports = {
-    methods: {
-      downloadCSV() {
-        window.open(Kolibri.urls['contentsummarylogcsv-list'](), '_blank');
-      },
-    },
     components: {
       'icon-button': require('icon-button'),
     },
+    computed: {
+      summarylogurl() {
+        return Kolibri.urls['contentsummarylogcsv-list']();
+      },
+      sessionlogurl() {
+        return Kolibri.urls['contentsessionlogcsv-list']();
+      },
+    },
   };
 
 </script>
 
 
-<style lang="stylus" scoped></style>
+<style lang="stylus" scoped>
+
+  @require 'jeet'
+
+  @media (min-width: 900px)
```

Comment:
I tried, and it's *almost* OK, but it allows it to get a bit too narrow before switching to vertical layout... not sure if we have a "large-breakpoint" or something I could use instead?

![image](https://cloud.githubusercontent.com/assets/618416/17910523/0fb811a6-693e-11e6-916e-fc7e1dd5bb7d.png)